src/components/search/LiveTVSearchResults.tsx

```diff
@@ -79,46 +78,26 @@ const LiveTVSearchResults: FunctionComponent<LiveTVSearchResultsProps> = ({ serv
             // Movies row
             fetchItems(apiClient, {
                 IncludeItemTypes: 'LiveTvProgram',
-                IsMovie: true,
-                IsSeries: false,
-                IsSports: false,
-                IsKids: false,
-                IsNews: false
+                IsMovie: true
             }).then(result => setMovies(result.Items || []));
             // Episodes row
             fetchItems(apiClient, {
                 IncludeItemTypes: 'LiveTvProgram',
-                IsMovie: false,
-                IsSeries: true,
-                IsSports: false,
-                IsKids: false,
-                IsNews: false
+                IsSeries: true
```

Comment:
After testing this, it may make sense to keep the exclusions for this one to avoid duplication in this row also... :man_shrugging:

Reply:
That's only the case until jellyfin/Jellyfin.XmlTv#12 hits in the next release of the XmlTv library. There is currently a bug that causes literally everything to identify as a series.

Reply:
Is XmlTv used for SchedulesDirect EPG data also?

Reply:
Glancing over the code it looks like it does not. I haven't used SchedulesDirect so I can't speak to how it's results will behave for search.

Reply:
I do use SD for guide data. That is where I was seeing the duplicates in the Episodes row, so it may need a similar fix. :slightly_smiling_face:

Reply:
Ok, that makes sense. Pushed that fix.